### ISSUE TITLE
Doesn't work.

### ISSUE THREAD

**Issue by Alex-Programs** [cyberboysumanjay/Torrent-Searcher#1]
The rest return no results.

**Comment by cyberboysumanjay** [comment]
Yeah, probably the domain is now blocking the access or things might have changed on the site.

**Comment by Alex-Programs** [comment]
OK. Thanks for writing it anyway!

**Comment by nitish771** [comment]
BTW. None of them is working now. So I'll try to modify the codes for the optimal use.
